libnavigation-base/src/main/java/com/mapbox/navigation/base/network/replay/ReplayRouteLocationConverter.kt

```diff
@@ -0,0 +1,143 @@
+package com.mapbox.navigation.base.network.replay
+
+import android.location.Location
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.geojson.LineString
+import com.mapbox.geojson.Point
+import com.mapbox.turf.TurfConstants
+import com.mapbox.turf.TurfMeasurement
+import java.util.ArrayList
+
+internal class ReplayRouteLocationConverter(
```

Comment:
Replay location engine shouldn't live in the `base`, but in the `core` or even in the `ui` module. You can always inject the engine via the constructor.

Reply:
suggest to move all replay classes to `Core`(IMO `UI` is not best place because it's not a part of Views).